main/glossary/README.md

```diff
@@ -97,20 +149,29 @@ An *object capability system* constrains how references are obtained. You can't
 If references can only be obtained by creation, construction, or introduction, you may have a safe system. If they can be obtained in any other way, your system is unsafe.
 
 For more information, see [Douglas Crockford on Object Capabilities](https://frontendmasters.com/courses/good-parts-javascript-web/object-capabilities/).
-
+<a name="Payments"></a>
 ## Payment
-An [AssetHolder](#assetholder). Payments hold amounts of certain rights issued by Mints, specifically amounts that are in _transit_. Amounts from payments can be deposited in purses, but otherwise, the entire amount is available when the payment is transferred. Payments can be converted to Purses.
+Payments hold [amounts](#Amounts) of certain rights **(tyg todo: I'd like to change "rights" to "assets". OK?)** 
+issued by [Mints](#Mints). Specifically amounts that are *in transit* from one party to another. 
+Amounts from payments can be deposited in [purses](#Purses), but otherwise, the entire amount is 
+available when the payment is transferred. Payments can be converted to [Purses](#Purses). All contents 
```

Comment:
```suggestion
Amounts from payments can be deposited in [purses](#Purses). A payment must be used up in its entirety by being deposited, split into smaller payments or combined into a larger payment. Payments can be converted to [Purses](#Purses). All contents 
```

I know this was in the original but it sounds very weird to me on rereading. Maybe take out the second part?